WSL/troubleshooting.md

```diff
@@ -36,6 +36,39 @@ Lastly, if your issue is related to the Windows Terminal, Windows Console, or th
 
 ## Common issues
 
+### `command not found` when executing windows .exe in linux
+Users can run Windows executables like notepad.exe directly from Linux. Sometimes, you may hit "command not found" like below: 
+```
+$ notepad.exe
+-bash: notepad.exe: command not found
+```
+
+If there are no win32 paths in your $PATH, interop isn't going to find the .exe.
+You can verify it by running `echo $PATH` in Linux. It's expected that you will see a win32 path (for example, /mnt/c/Windows) in the output.
+If you can't see any Windows paths then most likely your PATH is being overwritten by your Linux shell. 
+
+Here is a an example that /etc/profile on Debian contributed to the problem:
+```
+if [ "`id -u`" -eq 0 ]; then
+  PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+else
+  PATH="/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games"
+fi
+```
+
+The correct way on Debian is to remove above lines.
+You may also append $PATH during the assigment like below, but this lead to some [other problems](https://salsa.debian.org/debian/WSL/-/commit/7611edba482fd0b3f67143aa0fc1e2cc1d4100a6) with WSL and VSCode..
```

Comment:
Can you remove the lines after this? We should give them one clear solution, which in this case is should be just to remove the lines of the shell that sets the Path. Once done we can merge this in!